src/support.cpp

```diff
@@ -892,9 +892,9 @@ void AreaSupport::generateSupportAreasForMesh(SliceDataStorage& storage,
                                            if (layer_idx > 1)
                                            {
                                                // shrink a little so that areas that only protrude very slightly are ignored
```

Comment:
This comment needs to be removed; the shrinking logic in the next line is removed
```suggestion
```